src/Microsoft.ServiceFabric.Services.Wcf/Communication/Wcf/Client/WcfExceptionHandler.cs

```diff
@@ -177,7 +177,7 @@ public WcfExceptionHandler()
                     e,
                     false,
                     retrySettings,
-                    retrySettings.DefaultMaxRetryCount);
+                    retrySettings.DefaultMaxRetryCountForNonTransientErrors);

```

Comment:
>DefaultMaxRetryCountForNonTransientErrors [](start = 34, length = 41)

Transient error

Reply:
This is non-transient error , hence using non transient retry count.